### ISSUE TITLE
namespace: Nextcloud

### ISSUE THREAD

**Issue by aalaesar** [ansible/galaxy#3122]
Hello there! Nextcloud is getting an ansible collection and needs a galaxy namespace to publish it.
Thank you in advance.
## Namespace Request
Namespace: Nextcloud

##### Description
📱☁️💻 A safe home for all your data – community-driven, free & open source 👏

##### GitHub Org Link
https://github.com/nextcloud

##### Admins
@andyscherzinger
@aalaesar
@staticdev
@wiktor2200

if the users exists:
@blizzz
@mgallien

**Comment by thedoubl3j** [comment]
namespace created and admins @AndyScherzinger @aalaesar and @staticdev added. 

Users @wiktor2200 @blizzz and @mgallien did not return as valid usernames. If they have logged into galaxy prior to our github username self assignment, we will need their legacy usernames to add them. They can find that username by logging into galaxy authenticating with github. Once that is done, their username will be in the top right. 

Current admins can also add them, we (Galaxy admins) are not needed for that unless the users cannot be located. Let us know if we can help with any thing else :). (ps we <3 nextcloud)

**Comment by AndyScherzinger** [comment]
👍 Nice work 🎉

**Comment by mgallien** [comment]
@thedoubl3j I have now logged in to galaxy via my github account
so @mgallien should now exist
thanks

**Comment by AndyScherzinger** [comment]
Added you @mgallien 👍

**Comment by wiktor2200** [comment]
Hi! I've just logged in to galaxy via Github, so my account should also exist now.

**Comment by aalaesar** [comment]
Added you @wiktor2200 😃
